scripts/api_dump_generator.py

```diff
@@ -618,12 +618,28 @@
 
 //========================= Basetype Implementations ========================//
 
-@foreach basetype
+@foreach basetype where(not '{baseName}' in ['ANativeWindow', 'AHardwareBuffer', 'CAMetalLayer'])
```

Comment:
Should this be based instead of the POINTER_TYPES list instead of hardcoding it?

Reply:
POINTER_TYPES is not available in the middle of the *_CODEGEN strings.